### DISCUSSION TITLE
mysqli::real_connect(): (HY000/3118): Access denied for user 'root'@'localhost'. Account is locked.

### DISCUSSION THREAD

**Discussion by Nikolay-Dimitrov-Dimitrov** [leokhoa/laragon#637] [Q&A]
Good day

In phpMyAdmin I accidentally pressed on root lock button.
Please assist. I can't enter the database.
![Uploading Screenshot 2023-12-11.png…]()

Thank you

**Comment by Daviddjire** [comment]
in yourCLI

mysql -u root -p

mysql> SELECT user, host, account_locked FROM mysql.user where user = 'root';

ALTER USER [root@localhost](mailto:username@localhost) ACCOUNT UNLOCK;

**Reply by Md123etplus** [reply]
This works for me, thx!

**Reply by amangat2002** [reply]
how can you please tell me i'm facing same difficulty

**Reply by moamahfouz** [reply]
I run the same command ``` mysql -u root -p ```
And I got

``` ERROR 3118 (HY000): Access denied for user 'root'@'localhost'. Account is locked. ```

**Reply by hakimisidqi** [reply]
what is CLI
